pipelines/HSC/RC2.yaml

```diff
@@ -0,0 +1,340 @@
+description: The DRP pipeline specialized for the HSC RC2 dataset.
```

Comment:
Consider including pipeline name in files (e.g., DRP-RC2), because there are pipelines other than DRP in `drp_pipe` (e.g., `QaTractTables`).